services/API-service/src/api/event/event.service.ts

```diff
@@ -493,11 +495,14 @@ export class EventService {
         'disaster."mainExposureIndicator" as "exposureIndicator"',
         'event."mainExposureValue" as "exposureValue"',
         `CASE
+        WHEN event."userTrigger" = true THEN 'Trigger'
```

Comment:
fine for now, although this will quickly change again with the calculation of 'trigger' based on 'userTrigger' and 'forecastTrigger'